app/controllers/api/settings_controller.rb

```diff
@@ -0,0 +1,39 @@
+module API
+  class SettingsController < BaseController
+    def update
+      permitted = params.require(:user).permit(
+        :name, :location, :bio,
+        pronoun_parts: []
```

Comment:
What does this line do? Would it reset the pronoun parts?

Reply:
This is telling Rails to expect `params[:pronoun_parts]` to be an array, and cope with that.

Reply:
Aha, okay